config/mainnet/relayerExternalInventory.json

```diff
@@ -33,7 +33,6 @@
     "USDT": {
       "1135": {
         "targetPct": 30,
-        "thresholdPct": 10,
```

Comment:
I believe this change was committed unintentionally. You'll require to revert this change to let the CI pass.

Reply:
I've intentionally included a faulty configuration that PR pipeline fails that you may check how it works.